packages/bull-vault/test/integration-test/AuctionBullTestFork.t.sol

```diff
@@ -487,4 +1366,31 @@ contract AuctionBullTestFork is Test {
         );
         assertEq(bullStrategy.getCrabBalance().sub(crabToBeMinted), bullCrabBalanceBefore);
     }
+
+    function _calcTargetCollateralAndDebtInLeverage() internal view returns (uint256, uint256) {
+        uint256 ethUsdPrice = UniOracle._getTwap(
+            controller.ethQuoteCurrencyPool(),
+            controller.weth(),
+            controller.quoteCurrency(),
+            TWAP,
```

Comment:
this should prb be period 1 unless you always make sure that you wait at least TWAP seconds between transactions